clang-tools-extra/test/clang-doc/single-source-html.cpp

```diff
@@ -0,0 +1,2 @@
+// RUN: clang-doc --format=html --executor=standalone %s -output=%t/docs | FileCheck %s
+// CHECK: Using default asset: {{.*}}{{[\/]}}share{{[\/]}}clang
```

Comment:
A test that only tests for logspam doesn't seem very useful. I would've expected a change adding an `--asset` option to add a test that uses that option.

Reply:
I added a much more comprehensive test now I think